packages/next/src/server/use-cache/use-cache-wrapper.ts

```diff
@@ -710,7 +716,7 @@ export function cache(
         let entry = shouldForceRevalidate(workStore, workUnitStore)
           ? undefined
           : 'getExpiration' in cacheHandler
-            ? await cacheHandler.get(serializedCacheKey)
+            ? await cacheHandler.get(serializedCacheKey, { displayName })
```

Comment:
I think this is where we maybe need to go the manifest route rather than instrument the cache function itself. If we put this here we are forcing this to be the interface for cache handlers basically forever.

I have lots of bad ideas about what to do but no good ones yet